lighthouse-core/runner.js

```diff
@@ -254,7 +254,8 @@ class Runner {
 
       Sentry.captureException(err, {tags: {audit: audit.meta.name}, level: 'error'});
       // Non-fatal error become error audit result.
-      return Audit.generateErrorAuditResult(audit, 'Audit error: ' + err.message);
+      const debugString = err.friendlyMessage || `Audit error: ${err.message}`;
```

Comment:
I think we should add the err.code at the end of the debugString so we can differentiate the various errors with identical messages.

Reply:
Alright, fine with me